ci/bitrise-setup-readme.md

```diff
@@ -0,0 +1,100 @@
+# Nodes iOS Bitrise - Setup Guide 🛠
+
+## Xcode
+
+Make sure your project's schemes are ***shared (including app extensions)*** , otherwise you will run into a lot of warnings and it might not build all of your targets.
+
+
+## Bitrise
+
+1) Create a *New App* from the Bitrise dashboard
+- Make sure you choose **Nodes** account when creating the *New App* and make it **Private**
+
+<p align="center">
+  <img src="../images/ci/01-new-app-bitrise.jpg?raw=true" alt="New App"/>
+</p> 
+
+2) Connect the repository to Bitrise
+
+<p align="center">
+  <img src="../images/ci/02-new-app-bitrise.jpg?raw=true" alt="New App"/>
+</p> 
+
+3) When setting up the access make sure to auto-add SSH key
+
+<p align="center">
+  <img src="../images/ci/03-new-app-bitrise.jpg?raw=true" alt="New App"/>
+</p> 
+
+4) Indicate which branch Bitrise should use to make the builds
+
+<p align="center">
+  <img src="../images/ci/04-new-app-bitrise.jpg?raw=true" alt="New App"/>
+</p> 
+
+5) After choosing the branch, Bitrise will validate the source code and then you can select the *export method* you want to use. Usually is **app-store**
+
+<p align="center">
+  <img src="../images/ci/05-new-app-bitrise.jpg?raw=true" alt="New App"/>
+</p> 
+
+6) The last step is the Webhook setup. It will create a link from the github repository to Bitrise.
+
+<p align="center">
+  <img src="../images/ci/06-new-app-bitrise.jpg?raw=true" alt="New App"/>
+</p> 
+
+
+## Workflow
```

Comment:
Everything from here below I think we should move into the https://github.com/nodes-ios/bitrise-step-nodes-custom-script repository as it is describing what is there and then just link there from here.